src/deadline/client/cli/_groups/queue_group.py

```diff
@@ -268,19 +268,21 @@ def queue_get(**args):
     default=False,
 )
 @_handle_error
-@api.record_success_fail_telemetry_event(metric_name="incremental_output_download")
-def incremental_output_download(
+@api.record_success_fail_telemetry_event(metric_name="queue_sync_output")
+def sync_output(
     json: bool,
     bootstrap_lookback_minutes: float,
     checkpoint_dir: str,
     force_bootstrap: bool,
     ignore_storage_profiles: bool,
+    conflict_resolution: str,
     dry_run: bool,
     **args,
 ):
     """
-    BETA - Downloads job attachments output incrementally for all jobs in a queue. When run for the
-    first time or with the --force-bootstrap option, it starts downloading from --bootstrap-lookback-minutes
+    BETA - Downloads any new job attachments output for all jobs in a queue since the last run of the same command.
+
+    When run for the first time or with the --force-bootstrap option, it starts downloading from --bootstrap-lookback-minutes
     in the past. When run each subsequent time, it loads  the previous checkpoint and continues
```

Comment:
```suggestion
    in the past. When run each subsequent time, it loads the previous checkpoint and continues
```

nit: there's a double space, not from your change but just caught my eye